seedlet/assets/sass/vendors/jetpack/blocks/blog-posts/_editor.scss

```diff
@@ -146,6 +145,51 @@
 	}
 }
 
+// Block Style
+
+@include media(tablet) {
+	.wp-block-a8c-blog-posts.is-style-seedlet-alternating-grid {
+
+		// Necessary so that the block boundaries are respected.
+		overflow: hidden;
+
+		h2 {
+			margin-left: calc(50% + (0.5 * var(--global--spacing-horizontal)));
```

Comment:
Should this be applied as an `nth-of-type` rule?

This is what the headings look like to me in the editor and front-end: 

<img width="682" alt="Screen Shot 2020-08-26 at 8 13 16 AM" src="https://user-images.githubusercontent.com/5375500/91302289-3a62d680-e774-11ea-8253-9e5cd87f809f.png">

Reply:
Hmm, that's weird — I didn't see that before. I'll re-test. If anything, I can probably make that a direct-child rule.

Reply:
I think I fixed this in 974b049.